src/engines/html5/html5.js

```diff
@@ -495,10 +495,12 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    */
   enterPictureInPicture(): void {
     try {
-      // Currently it's supported in chrome and in safari. So if we consider checking support before,
-      // we can use this flag to distinguish between the two. In the future we might need a different method.
-      // Second condition is because flow does not support this API yet
-      if (document.pictureInPictureEnabled && typeof this._el.requestPictureInPicture === 'function') {
+      if (typeof this._el.webkitSetPresentationMode === 'function') {
+        this._el.webkitSetPresentationMode('picture-in-picture');
```

Comment:
please move 'picture-in-picture' to const

Reply:
it's browser's string. we use it as is (like 'function', 'inline', 'playsinline' and many more)